.gitignore

```diff
@@ -3,3 +3,4 @@ _site/
 .jekyll-cache/
 .jekyll-metadata
 Gemfile.lock
+.DS_Store
```

Comment:
I'm fine with this but you might want to consider configuring your [global gitignore file](https://fabiorosado.dev/blog/global-gitignore/) so you don't need to do this for every project.

Reply:
I think this is a helpful change here. I'd say most people I know don't configure their global .gitignore file (I know I don't bother)